ykllvmwrap/src/jitbuilder.cc

```diff
@@ -0,0 +1,306 @@
+using namespace llvm;
+using namespace std;
+// An atomic counter used to issue compiled traces with unique names.
```

Comment:
probably need a new line between `using` and the following line.

Reply:
Fixed in 9f14d25.